pandas/_libs/tslib.pyx

```diff
@@ -152,7 +152,7 @@ def format_array_from_datetime(
         # a format based on precision
         basic_format = format is None
         if basic_format:
-            reso_obj = get_resolution(values, reso=reso)
+            reso_obj = get_resolution(values, tz=tz, reso=reso)
```

Comment:
This shouldn't affect any tests?

Reply:
correct; tz is None here, just making it explicitly passed